fictadvisor-web/src/pages/admin/teachers/edit/[teacherId].tsx

```diff
@@ -1,11 +1,16 @@
 import React from 'react';
+import { useRouter } from 'next/router';
 
 import AdminPanelLayout from '@/components/common/layout/admin-panel-layout/AdminPanelLayout';
+import TeachersAdminEditPage from '@/components/pages/admin/teachers-admin-page/pages/teachers-admin-edit-page';
 
 const TeacherId = () => {
+  const router = useRouter();
+  const teacherId = router.query.teacherId as string;
+  console.log(teacherId);
```

Comment:
Remove console log.
Update to App router and use new router and useSearchParams hook